tests/entities/conftest.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import random
+import uuid
+
+from tests.helper_functions import random_str, random_int
+
+from mlflow.entities import RunInfo, LifecycleStage, RunStatus
+from mlflow.entities import Metric, RunData, Param, RunTag
+from mlflow.utils.time_utils import get_current_time_millis
+
+
+@pytest.fixture(scope="module")
+def metrics():
+    yield [
+        Metric(
+            key=random_str(10),
+            value=random_int(0, 1000),
+            timestamp=get_current_time_millis() + random_int(-1e4, 1e4),
+            step=random_int(),
+        )
+    ]
+
+
+@pytest.fixture(scope="module")
+def params():
+    yield [Param(random_str(10), random_str(random_int(10, 35))) for _ in range(10)]
+
+
+@pytest.fixture(scope="module")
+def tags():
+    yield [RunTag(random_str(10), random_str(random_int(10, 35))) for _ in range(10)]
```

Comment:
I don't think we need to fixturize everything. If you don't need teardown, a plain function should normally suffice.